test/code-build-test.js

```diff
@@ -94,6 +94,21 @@ describe("githubInputs", () => {
       .and.to.deep.equal(["one", "two", "three", "four"]);
   });
 
+  it("skips override when parameter is set to true", () => {
```

Comment:
1. could you also add a assertion in [build basic parameters for codeBuild.startBuild](https://github.com/aws-actions/aws-codebuild-run-build/blob/master/test/code-build-test.js#L60) that expect(test).to.haveOwnProperty("disableSourceOverride").and.to.equal(false);

2. the test case name is a bit confusing. should we just called it `"can process disable-source-override"`